src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGalleryListView.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+
+import {
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  DataListAction,
+  Flex,
+  FlexItem,
+  Label,
+  LabelGroup,
+} from '@patternfly/react-core';
+import OutlinedQuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+import { useTranslation } from '@rhoas/app-services-ui-components';
+
+export type ConnectorTypesListItemProps = {
+  id: string;
+  labels: string[];
+  title: string;
+  verison: string;
+};
+
+export const ConnectorTypeListItem: React.FunctionComponent<ConnectorTypesListItemProps> =
```

Comment:
It'd be good to keep the file name consistent with the main function component name, probably `ConnectorTypesListItem.tsx` I guess?